services/circleci/circleci.service.js

```diff
@@ -0,0 +1,106 @@
+'use strict'
+
+const Joi = require('joi')
+const BaseJsonService = require('../base-json')
+
+const circleSchema = Joi.array()
+  .items(Joi.object({ status: Joi.string() }))
+  .min(1)
```

Comment:
I _think_ this is a sensible rule. I can't see a case for treating `[]` as a valid response here..